docs/src/examples/components/Divider/Rtl/DividerExample.rtl.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react'
+import { Divider } from '@stardust-ui/react'
+
+const DividerExampleShorthand = () => <Divider content="مثال النص" />
+
+export default DividerExampleShorthand
```

Comment:
```suggestion
export default DividerExampleRtl
```

May be name should be changed?